changelogs/fragments/cloud_content_action_plugins.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
Can we remove all the intermediate changelogs, and add just one (minor_changes) reflecting the newly added scaffolding capabilities for cloud_content?